src/modules/cmdpal/Microsoft.CmdPal.UI/Helpers/ColorExtensions.cs

```diff
@@ -0,0 +1,398 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI;
+
+// IDE0047 (remove redundant parentheses) clashes with stylecop rules
+#pragma warning disable IDE0047
+
+namespace Microsoft.CmdPal.UI.Helpers;
+
+/// <summary>
+/// Extension methods for <see cref="Color"/>.
+/// </summary>
+internal static class ColorExtensions
+{
+    private const float ByteMax = byte.MaxValue;
+
+    public static float CalculateBrightness(this Color color)
+    {
+        var (_, _, brightness) = color.ToHsv();
+        return brightness;
+    }
+
+    /// <summary>
+    /// Allows to change the brightness by a factor based on the HSV color space.
+    /// </summary>
+    /// <param name="color">Input color.</param>
+    /// <param name="factor">The value of the brightness change factor from <see langword="100"/> to <see langword="-100"/>.</param>
+    /// <returns>Updated color.</returns>
+    public static Color UpdateBrightness(this Color color, float factor)
+    {
+        ArgumentOutOfRangeException.ThrowIfGreaterThan(factor, 100f);
```

Comment:
💭 it is a little curious that we're storing SL on a 0,100 scale rather than 0.0,1.0. Seems like there's a lot of places where we're multiplying or dividing by 100 somewhere.